### PR TITLE
folded together docs for transfer API

### PR DESCRIPTION
**Pull request by craigsteffen**
Transfer API doc

### Description

I'd previously created a documentation branch for the transfer API 
off the main repo by mistake.  As people inserted suggestions 
and edits, it got very messy.  I blew away that branch and started
this one instead.  This is where we can hammer out the last
edits and details of this initial documentation step.

### Motivation and Context

Transfer API isn't documented.  

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [ ] My code follows the UnifyFS code style requirements.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commit messages are properly formatted.

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -141,3 +141,59 @@ One form of running this example could be:
 .. explicit external hyperlink targets
 
 .. _examples: https://github.com/LLNL/UnifyFS/tree/dev/examples/src
+
+<<<<<<< HEAD
```
Comment (MichaelBrim) [inline docs/examples.rst]: You've got some leftover cruft from a conflict here

File: docs/examples.rst [hunk 2]
```diff
@@ -141,3 +141,59 @@ One form of running this example could be:
...
+    }
+    ...
+    out:
+    MPI_Finalize();
+
+    return ret;
+
+This code fragment is from the code example.c in the UnifyFS examples directory.
+=======
+Transfer API
```
Comment (MichaelBrim) [inline docs/examples.rst]: Rest of the conflict cruft starts here at `=======` and ends below at `>>>>>>> ...`.

**Comment by craigsteffen** [comment]
Hmm.  I'd forgotten about this.  I'll try to clean this up this week, or else cancel it.

**Comment by craigsteffen** [comment]
Not relevant at this point.  I'll circle back to documentation after the milestones are done.

**Pull request closed without merging**